lms/djangoapps/courseware/models.py

```diff
@@ -26,6 +26,7 @@
 from django.utils.translation import ugettext_lazy as _
 from model_utils.models import TimeStampedModel
 from opaque_keys.edx.django.models import BlockTypeKeyField, CourseKeyField, UsageKeyField
+from coursewarehistoryextended.fields import UnsignedBigIntAutoField
```

Comment:
The right thing to do is probably to refactor this generic field type out of a `coursewarehistoryextended` specific location.

Reply:
Done